yourgov/pages/app/licences-and-permits/index.tsx

```diff
@@ -2,160 +2,81 @@ import { Fragment, ReactElement } from 'react';
-												</TableRow>
-											</TableHead>
-											<TableBody>
-												{mockCurrentApprovals.map((approval, index) => (
-													<TableRow key={index}>
-														<TableCell as="th" scope="row" fontWeight="bold">
-															<TextLink href="/not-found">
-																{approval.approvalNumber}
-															</TextLink>
-														</TableCell>
-														<TableCell>{approval.type}</TableCell>
-														<TableCell>
-															<StatusBadge
-																appearance="subtle"
-																{...approval.status}
-															/>
-														</TableCell>
-														<TableCell>{approval.approvedDate}</TableCell>
-													</TableRow>
-												))}
-											</TableBody>
-										</Table>
-									</TableWrapper>
-								</Stack>
-							</TabPanel>
-							<TabPanel>
-								<Stack gap={1.5}>
-									<H2 id="access-requests-heading">Application history</H2>
-									<TableWrapper>
-										<Table aria-labelledby="access-requests-heading">
-											<TableHead>
-												<TableRow>
-													<TableHeader scope="col" width="25%">
-														Reference number
-													</TableHeader>
-													<TableHeader scope="col" width="25%">
-														Application type
-													</TableHeader>
-													<TableHeader scope="col" width="25%">
-														Application status
-													</TableHeader>
-													<TableHeader
-														scope="col"
-														textAlign="right"
-														width="25%"
-													>
-														Actions
-													</TableHeader>
-												</TableRow>
-											</TableHead>
-											<TableBody>
-												{mockApplicationHistory.map((application, index) => (
-													<TableRow key={index}>
-														<TableCell as="th" scope="row" fontWeight="bold">
-															<TextLink href="/not-found">
-																{application.referenceNumber}
-															</TextLink>
-														</TableCell>
-														<TableCell>{application.type}</TableCell>
-														<TableCell>
-															<StatusBadge
-																appearance="subtle"
-																{...application.status}
-															/>
-														</TableCell>
-														<TableCell textAlign="right">
-															<TextLink href="/not-found">View</TextLink>
-														</TableCell>
-													</TableRow>
-												))}
-											</TableBody>
-										</Table>
-									</TableWrapper>
-								</Stack>
-							</TabPanel>
-						</TabPanels>
-					</Tabs>
+					<ButtonLink
+						alignSelf="start"
+						href="/app/licences-and-permits/apply/mobile-food-vendor-permit"
+					>
+						Apply for a permit
+					</ButtonLink>
+					<ApplicationHistory />
+
+					<Divider />
+
+					<Stack gap={1.5}>
+						<H2>Popular links</H2>
+						<Columns as="ul" cols={{ xs: 1, sm: 2 }}>
+							<Card as="li" shadow clickable>
+								<CardInner>
+									<Stack gap={1}>
+										<Heading as="h2" type="h3">
```

Comment:
This renders as an `h2` but I think it should be `h3`.

Reply:
![image](https://github.com/user-attachments/assets/14171f67-ba8c-4538-8dc1-30d844d53ddf)

Reply:
Good pickup. New commit fixes this and other stuff